src/emitters/emitter.ts

```diff
@@ -14,20 +14,20 @@ import {
   Queue,
   Topic,
 } from "../types";
-import { Logger } from "../utils/utils";
 import { SqnsEmitter } from "./emitter.sqns";
+import { Logger } from "../utils/utils";
 
 export class Emitter implements IEmitter {
   private localEmitter: EventEmitter = new EventEmitter();
   private emitter!: IEmitter;
   private options!: IEmitterOptions;
+  private logger;
```

Comment:
Can add the logger interface type that you added

Reply:
Type is being automatically inferred from the constructor but we can add it for completeness

Reply:
Done.